molecule/builder-xenial/tests/test_security_updates.py

```diff
@@ -1,16 +1,31 @@
 import os
+from subprocess import check_output
+import re
+import pytest
+
 SECUREDROP_TARGET_PLATFORM = os.environ.get("SECUREDROP_TARGET_PLATFORM")
 testinfra_hosts = [
         "docker://{}-sd-sec-update".format(SECUREDROP_TARGET_PLATFORM)
 ]
 
 
+def is_rc_version():
+    command = "git describe --all"
+    version = check_output(command, shell=True).\
```

Comment:
we don't need to use the shell here if we use `check_output(["git", "describe", "--all"])`

Reply:
:+1: